modules/configure-an-additional-clientCA.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * security/certificates/api-server.adoc
+
+:_content-type: PROCEDURE
+[id="configure-an-additional-clientCA-for-the-OpenShift-API-server_{context}"]
+
+= Configure an additional clientCA for the OpenShift API server
+
+Optionally, you may choose to invalidate the installer-generated kubeconfig. You would do this when:
+* You don't trust who installed the cluster
```

Comment:
This list is wrongly formatted somehow, so it doesn't look well in the preview

Reply:
Seems to be OK now.